.github/autobuild/android.sh

```diff
@@ -23,6 +23,7 @@ QT_VERSION=6.3.2
 QT_BASEDIR="/opt/Qt"
 BUILD_DIR=build
 ANDROID_NDK_HOST="linux-x86_64"
+ANDROID_NDK_ROOT="${ANDROID_NDK_ROOT}"
```

Comment:
Why do this?

Reply:
Purely to get past the shellcheck error. It's otherwise totally unnecessary.

Reply:
@hoffie any idea why it's complaining?

Reply:
It complains because shellcheck is somewhat brittle and expects all environment variables to be explicitly assigned a value within the context of the script. 
However it is already set by the Github CI, so I re-assign the value in the script to side-step the error. That's it.

Reply:
Added explanatory comment for ANDROID_NDK_ROOT.
Also pushed some clang-format fixes.